### ISSUE TITLE
[CLOSED] gradido proxy

### ISSUE THREAD

**Issue by ulfgebhardt** [Zahlungsmittel/Zahlungsmittel#1353]
<a href="https://github.com/einhornimmond"><img src="https://avatars.githubusercontent.com/u/67117805?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [einhornimmond](https://github.com/einhornimmond)**
_Sunday Jan 16, 2022 at 18:14 GMT_
_Originally opened as https://github.com/gradido/gradido/pull/1294_

----

## 🍰 Pullrequest
A nginx config template for a gradido domain proxy. 

Example: we have the domain gdd1.gradido.com as or main domain but our server will be shutdown sometime and we like easly switch over to or backup server without waiting that all dns server has get the update for the new domain. 
The solution: We use nginx as proxy and route the gdd1.gradido.com from dns to our final production server.
The final production server then proxy all calls to our current production server which get a second domain: gdd1-1fire.gradido.com.
If we swap to the backup server we need only update the nginx config on the final production server and don't need to wait on dns server updates.

If we copy over the ssl keys from production to the final production server, we can proxy from final production to current production and keep the config on current production. So all user can reach the production server nevertheless if there request go to current production ip or final production server ip.

----

_**[einhornimmond](https://github.com/einhornimmond)** included the following code: https://github.com/gradido/gradido/pull/1294/commits_

**Comment by ulfgebhardt** [comment]
<a href="https://github.com/ulfgebhardt"><img src="https://avatars.githubusercontent.com/u/1238238?v=4" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [ulfgebhardt](https://github.com/ulfgebhardt)**
_Friday Jan 28, 2022 at 13:53 GMT_

----

closing as no longer needed
